sdk/identity/azure-identity/src/managed_identity_source.cpp

```diff
@@ -449,16 +449,27 @@ std::unique_ptr<ManagedIdentitySource> ImdsManagedIdentitySource::Create(
       credName + " will be created" + WithSourceMessage("Azure Instance Metadata Service")
           + ".\nSuccessful creation does not guarantee further successful token retrieval.");
 
-  return std::unique_ptr<ManagedIdentitySource>(new ImdsManagedIdentitySource(clientId, options));
+  std::string imdsHost = "http://169.254.169.254";
+  std::string customImdsHost = Environment::GetVariable("AZURE_IMDS_CUSTOM_AUTHORITY_HOST");
```

Comment:
Check if there's something that we can borrow for the identity's Readme.MD, if other SDKs document it in theirs (and the same for #5832)

Reply:
The other SDKs don't document/heavily advertise this capability.

Reply:
For #5832, addressed in https://github.com/Azure/azure-sdk-for-cpp/pull/5838